tests/dir.props

```diff
@@ -29,14 +32,23 @@
     <PackagesDir>$(ProjectDir)..\packages\</PackagesDir>
     <ToolsDir Condition="'$(ToolsDir)'==''">$(ProjectDir)..\Tools\</ToolsDir>
     <DotnetCliPath Condition="'$(DotnetCliPath)'==''">$(ToolsDir)dotnetcli/</DotnetCliPath>
+    <OverrideToolHost>$(DotnetCliPath)dotnet</OverrideToolHost>
     <BuildToolsTaskDir Condition="'$(BuildToolsTargets45)' == 'true'">$(ToolsDir)net46/</BuildToolsTaskDir>
+    <CSharpCoreTargetsPath Condition="'$(OS)' != 'Windows_NT'">$(ToolsDir)\Microsoft.CSharp.Core.targets</CSharpCoreTargetsPath>
+    <_TargetFrameworkDirectories Condition="'$(OS)' != 'Windows_NT'">$(MSBuildThisFileDirectory)/Documentation</_TargetFrameworkDirectories>
```

Comment:
Because someone has changed msbuild warning warning "MSB3644" to error and it getting failed because we do not have .NET Framework on Linux.